classes/Kohana/Jam/Range.php

```diff
@@ -59,13 +59,13 @@ public function __construct($source = NULL, $format = NULL)
 	{
 		if (is_string($source) AND $source !== '')
 		{
-			$source = explode('|', $source);
+			$source = array_map('intval', explode('|', $source));
 		}
 
 		if (is_array($source))
 		{
-			$this->min($source[0]);
-			$this->max($source[1]);
+			$this->min((int) $source[0]);
```

Comment:
@phgeorgiev 
Why i have to walk through array and add additional complexity when there is easier way to cast parameter  on assigning?